packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -122,11 +126,20 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   const [downloadErrorMessage, setDownloadErrorMessage] = React.useState('');
 
   const adapter = useAdapter();
-
   useEffect(() => {
     // Initial data should be always fetched by the composite(or external caller) instead of the adapter
     adapter.fetchInitialData();
   }, [adapter]);
+  /* @conditional-compile-remove(file-sharing) */
+  const locale = useLocale();
+  /* @conditional-compile-remove(file-sharing) */
+  const fileDownloadButtonString = useMemo(() => locale.strings.chat.downloadFileButton, [locale.strings]);
```

Comment:
these don't need to be memoized :)
useLocale will only return new objects if underlying strings change.

Reply:
done